trulens/nn/backend/__init__.py

```diff
@@ -77,7 +84,6 @@ def get_backend(suppress_warnings=False):
 _ALL_BACKEND_API_FUNCTIONS = [
     'dim_order',
     'channel_axis',
-    'Tensor',
```

Comment:
I can put this back in if we think that's a good idea. My thought is it should be an internal type (since tensorflow has several tensor types, not just one), so removing it is the "right" thing to do. But on the other hand it does change the backend interface to remove it.